.github/renovate.json

```diff
@@ -1,13 +1,16 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "configMigration": true,
 
   "extends": [
-    "config:base"
+    "config:recommended"
```

Comment:
Base was renamed to recommended in the last major: https://docs.renovatebot.com/release-notes-for-major-versions/#version-36